Front/src/app/services/shipment.service.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { AppConfig } from '../common/app-config';
+import { Observable } from 'rxjs';
+import { HttpResponseModel } from '../models/HttpResponseModel';
+import { ShipmentViewModel } from '../models/shipmentViewModel';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ShipmentService {
+
+  constructor(private http: HttpClient) { }
+  ServiceEndpoint: string = `${AppConfig.API_URL}`;
```

Comment:
On the service endpoint you can also concatenate the controller name so that you don't repeat it on every method. 

```suggestion
  ServiceEndpoint: string = `${AppConfig.API_URL}/api/Shipment`;
```